presto-docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -259,6 +259,77 @@ Field           Required  Type      Description
 
 There is no limit on field descriptions for either key or message.
 
+Row Encoding
+------------
+
+For key and message, an encoder is used to map message and key data onto table columns.
```

Comment:
A row encoder defines how Kafka key and message data onto table columns in Presto.

Reply:
Also need to add what encoding is for .. it allows SQL insert statements to be used and defines how data is encoded into the Kafka stream... or something along those lines